src/main/java/nus/climods/ui/common/CommandBox.java

```diff
@@ -30,8 +31,13 @@ public CommandBox(CommandSession commandSession) {
         super(FXML);
         this.commandSession = commandSession;
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
+        commandTextField.requestFocus();
         commandTextField.textProperty().addListener((unused1, unused2, unused3) -> setStyleToDefault());
         commandTextField.setOnKeyPressed(event -> {
+            if (event.getCode().equals(KeyCode.TAB)) {
+                commandTextField.requestFocus();
+            }
```

Comment:
The issue raised by @zupey could be related to https://stackoverflow.com/a/32807849

TLDR, events will not be passed to UI nodes unless they are focused